waltz-tools/src/test/java/com/wepay/waltz/tools/cluster/ClusterCliTest.java

```diff
@@ -184,6 +184,10 @@ public void testVerifyCommand() throws Exception {
             assertTrue(outContent.toString("UTF-8")
                 .contains("Validation PARTITION_QUORUM_STATUS failed for partition 2"));
 
+            // Check that STORAGE_RECOVERY_COMPLETE didn't fail
+            assertFalse(outContent.toString("UTF-8").contains("Recovery not completed"));
+            assertFalse(outContent.toString("UTF-8").contains("Cannot obtain replica states"));
+
```

Comment:
Didn't find a good way to add unit test for the scenario when recovery is ongoing and this verification fails, so I just added here to check STORAGE_RECOVERY_COMPLETE is successful in normal case. Suggestions are welcomed on how to do the testing.